docs/dev.md

```diff
@@ -62,3 +64,22 @@ Integration tests can read the following system properties to obtain information
 | liberty.hostname | The host name of the Liberty server. |
 | liberty.http.port | The port used for client HTTP requests. |
 | liberty.https.port | The port used for client HTTP requests secured with SSL (https). |
+
+##### devc, Container Mode
+
+This is a technology preview.
```

Comment:
Also say something like "The features and parameters described below may change in future milestones or releases of the Liberty Maven plugin."

Reply:
Ok